WooCommerce/WooCommerceTests/System/SessionManagerTests.swift

```diff
@@ -190,6 +190,48 @@ class SessionManagerTests: XCTestCase {
         XCTAssertNil(defaults[UserDefaults.Key.usedProductDescriptionAI])
     }
 
+    /// Verifies that `hasDismissedWriteWithAITooltip` is set to `nil` upon reset
+    ///
+    func test_hasDismissedWriteWithAITooltip_is_set_to_nil_upon_reset() throws {
+        // Given
+        let uuid = UUID().uuidString
+        let defaults = try XCTUnwrap(UserDefaults(suiteName: uuid))
+        let sut = SessionManager(defaults: defaults, keychainServiceName: Settings.keychainServiceName)
+
+        // When
+        defaults[UserDefaults.Key.hasDismissedWriteWithAITooltip] = true
+
+        // Then
+        XCTAssertTrue(try XCTUnwrap(defaults[UserDefaults.Key.hasDismissedWriteWithAITooltip] as? Bool))
+
+        // When
+        sut.reset()
+
+        // Then
+        XCTAssertNil(defaults[UserDefaults.Key.hasDismissedWriteWithAITooltip])
+    }
+
+    /// Verifies that `numberOfTimesWriteWithAITooltipIsShown` is set to `nil` upon reset
+    ///
+    func test_numberOfTimesWriteWithAITooltipIsShown_is_set_to_nil_upon_reset() throws {
+        // Given
+        let uuid = UUID().uuidString
+        let defaults = try XCTUnwrap(UserDefaults(suiteName: uuid))
+        let sut = SessionManager(defaults: defaults, keychainServiceName: Settings.keychainServiceName)
+
+        // When
+        defaults[UserDefaults.Key.numberOfTimesWriteWithAITooltipIsShown] = true
```

Comment:
Should this be a number instead of a bool?

Reply:
Fixed in 75ec90468682b10885b12f07700001d850e69be0